EventCatalogAPI/Domain/CatalogEvent.cs

```diff
@@ -10,12 +10,17 @@ public class CatalogEvent
         public int EventId { get; set; }
         public string EventName { get; set; }
         public string Description { get; set; }
-        public string Location { get; set; }
+        public DateTime Date { get; set; }
         public string Address { get; set; }
-        public decimal Fee { get; set; }
+        public decimal Price{ get; set; }
         public string EventImageUrl { get; set; }
-        public string EventStartTime { get; set; }
-        public string EventEndTime { get; set; }
+        public float EventStartTime { get; set; }
+        public float EventEndTime { get; set; }
+        public int Duration { get; set; }
+        public int EventTypeId { get; set; }
+        public int EventCategoryId { get; set; }
+        public CatalogCategory CatalogCategory { get; set; }
```

Comment:
Why do we have line 22 and 23?

Reply:
I am trying to establish a relationship between all the models..In future the Event class can talk to Type and Category classes with the help of the foreign key(EventTypeId and EventCategoryId)

Reply:
I guess I'm mostly confused about the naming. What do you think about renaming the class as EventCategory? Trying to understand why it is called Catalog Category

Reply:
sure we could do that